package.json

```diff
@@ -1,11 +1,14 @@
 {
   "repository": {
     "type": "git",
-    "url": "https://github.com/ProtonMail/Angular.git"
+    "url": "https://github.com/ProtonMail/WebClient.git"
```

Comment:
I was trying to remember what was the name of the private repo. Now, I do. :)

BTW, I see that the version deployed to https://mail.protonmail.com is v3.11.2 from Sep 1st. The code here is at 3.11.0. Is the private repo still being used? It makes hard to make public contributions this way.

Reply:
:grin: voui. +1
We will try to update the repo.